### ISSUE TITLE
The importError when runing the generate sample script

### ISSUE THREAD

**Issue by Ding-Guo** [PeterWang512/GANSketching#6]
When I run generate sample script,there is a importError "ImportError: /home/dzl/.cache/torch_extensions/py38_cu113/fused/fused.so: cannot open shared object file: No such file or directory"

Library version:

python 3.8.12
torch 1.10.1
cuda 11.3
ninja 1.10.2

could you tell me what your library version?
thank you

**Comment by PeterWang512** [comment]
Seems related to this: https://github.com/rosinality/stylegan2-pytorch/issues/5
Can you try removing the cache folder and running the script again, as described by the attached post? Thanks!
In your case, maybe try removing the torch_extensions folder?

**Comment by Ding-Guo** [comment]
I have tried it again, but it didn't solve the problem, I think the trouble is the library version.  And when i open the directory of the "/home/dzl/.cache/torch_extensions/py38_cu113/fused/", there is only a file "build.ninja".

**Comment by PeterWang512** [comment]
Sure, here's mine:

python 3.7.6
torch 1.7.1
cuda 10.2
ninja 1.101

Hope this helps!

**Comment by Ding-Guo** [comment]
Thank you!

**Comment by Ding-Guo** [comment]
I tried it again, but there is still the problem, I want to ask if the program needs a cudnn.

**Comment by kwonminki** [comment]
Did you solve the problem...? I have the same issue... :(

**Comment by Ding-Guo** [comment]
No, I tried to change a lot of cuda and pytorch versions, but I still can't solve this problem under Ubuntu. The Ubuntu version I use is 20.04. Finally, I successfully run it on Windows system. The system I use is Windows 11, the gpu is RTX 3080, the cuda version is 11.3, and the nVidia graphics card driver version is 496.13.

**Comment by abdullah22842** [comment]
@Ding-Guo  Did you solve the problem? I have same issue...

python 3.7
GCC 7.5
Torch 1.10.1+cu102
Build cuda_11.4
ninja 1.11.1.
